src/styles/_header.scss

```
@@ -37,5 +37,12 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: 300ms;
```

```suggestion
    transition: color 300ms;
```
Завжди врато прописувати, ЯКА САМЕ властивість анімується